eng/pipelines/maintenance-packages.yml

```diff
@@ -96,7 +96,43 @@ stages:
               $(_InternalBuildArgs)
             displayName: $(_BuildJobDisplayName)
 
+- stage: pack
```

Comment:
Is it worth having a separate stage for this? I think it's easier to just have an additional job.

Reply:
That was a question I left above. Since you're also not sure, then I'll change to that.